library/alloc/src/vec/into_iter.rs

```diff
@@ -121,6 +121,11 @@ impl<T, A: Allocator> IntoIter<T, A> {
             ptr::drop_in_place(remaining);
         }
     }
+
+    /// Forgets to Drop the remaining elements while still freeing the backing allocation.
```

Comment:
The comment isn't correct. The method itself doesn't free the allocation since it only takes `&mut self`, dropping `IntoIter` later will do so.